src/utils/getAssetsFilters.jsx

```diff
@@ -31,3 +31,7 @@ export const getFilterStateObjectFromArray = (enabledFilters) => {
 
   return filterState;
 };
+
+export const hasSearchOrFilterApplied = (filters, search) => (
+  Boolean(hasSelectedFilters(filters) || (search && search.length > 0))
```

Comment:
Out of curiosity, why cast it to a `Boolean`?

Reply:
If `search` is set to something falsey, the expression will just return the value. Wrapping it in this cast will make sure it only ever returns `true` or `false`.

Reply:
Gotcha. Thanks!